install_config/certificate_customization.adoc

```diff
@@ -103,6 +103,62 @@ the server to pick up the configuration changes.
 
 For the master API or web console, wildcard names are accepted.
 
+[[configuring-custom-certificates-wildcard]]
+== Configuring a Custom Wildcard Certificate for the Default Router
+
+You can configure the {product-title} default router with a default wildcard certificate. A default wildcard certificate provides a convenient way for 
+applications that are deployed in {product-title} to use default encryption without needing custom certificates. 
+
+[NOTE]
+====
+Default wildcard certificates are recommended for non-production environments only.
+====
+
+In order to configure a default wildcard certificate, provision a certificate that is valid for `.<app_domain>`, where `<app_domain>` is the value of `openshift_master_default_subdomain` in the xref:../install_config/install/advanced_install.adoc#configuring-ansible[Ansible inventory file], by default `/etc/ansible/hosts`. Once provisioned, place the certificate, key, and ca certificate files on your Ansible host, and add the following line to your Ansible inventory file.
+
+----
+openshift_hosted_router_certificate={"certfile": "/path/to/apps.c1-ocp.myorg.com.crt", "keyfile": "/path/to/apps.c1-ocp.myorg.com.key", "cafile": "/path/to/apps.c1-ocp.myorg.com.ca.crt"}
+----
+
+For example:
+
+----
+openshift_hosted_router_certificate={"certfile": "/home/cloud-user/star-apps.148.251.233.173.nip.io.cert.pem", "keyfile": "/home/cloud-user/star-apps.148.251.233.173.nip.io.key.pem", "cafile": "/home/cloud-user/ca-chain.cert.pem"}
+----
+
+Where the parameter values are: 
+
+* *certfile* is the path to the file that contains the {product-title} router certificate.
+
+* *keyfile* is the path to the file that contains the {product-title} wildcard key.
+
+* *cafile* is the path to the file that contains the root CA for this key and certificate. If an intermediate CA is in use, the file should contain both the intermediate and root CA.
+
+If these certificate files are new to your {product-title} cluster, run the Ansible *_deploy_cluster.yml_* playbook to add these files to the {product-title} configuration files. 
+The playbook adds the certificate files to the *_/etc/origin/master/named_certificates/_* directory.
```

Comment:
The specified customized cert files will be uploaded to "/etc/origin/master/", not "/etc/origin/master/named_certificates/".

Reply:
Is this true in all versions or just 3.6?

Reply:
I think it is true in all version.

Reply:
Ok. I'm making a new PR to fix this path in 3.9 and 3.10: https://github.com/openshift/openshift-docs/pull/10076